.github/workflows/checks.yml

```diff
@@ -167,6 +169,23 @@ jobs:
       - name: Run tests
         run: npx nx run-many --target=e2e
 
+      - uses: actions/upload-artifact@v4
+        if: always()
+        id: upload-screenshots
+        with:
+          name: cypress-screenshots
+          path: |
+            apps/datahub-e2e/cypress/screenshots/**/*
+            apps/metadata-editor-e2e/cypress/screenshots/**/*
```

Comment:
I see this on the CI run when I try to look for the screenshots, is that normal?

![image](https://github.com/geonetwork/geonetwork-ui/assets/132347903/a19fc21e-0790-4893-b77c-5d4972d91843)

Reply:
you're right, I think if the previous runs are still in cache no screenshots will be produced; in that case the PR comment should not be modified

Reply:
This is addressed now 🙂